helpers/event_helper.rb

```diff
@@ -0,0 +1,11 @@
+module EventHelper
+  def event_time(evt_start, evt_end = nil, mobile_only: false)
```

Comment:
this abbreviation isn't really good. I would either use the full name (`event_start`), or simply `start`